### PR TITLE
{bio,numlib}[GCC/11.3.0] Beast v2.7.3, beagle-lib v4.0.0

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/b/Beast/Beast-2.7.3-GCC-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted 
+ proportional to its posterior probability. """
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+
+_archs = {'x86_64': 'x86', 'aarch64': 'aarch64'}
+
+source_urls = ['https://github.com/CompEvol/beast2/releases/download/v%(version)s/']
+sources = ['BEAST.v%%(version)s.Linux.%s.tgz' % _archs[ARCH]]
+checksums = ['234af8572e827c1c856f0e1ad46e61fa3b3b875c6b5d246f67839ec7c1bf40b5']
```
Comment (jfgrimm) [inline easybuild/easyconfigs/b/Beast/Beast-2.7.3-GCC-11.3.0.eb:20]: surely the different `_archs` will have different checksums?

**Review by jfgrimm — DISMISSED** [review]

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in total)
node067.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/bb2bd64c5e9411443473703ebe62c858 for a full test report.

File: easybuild/easyconfigs/b/beagle-lib/beagle-lib-4.0.0-GCC-11.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+sources = ['v%(version)s.tar.gz']
+checksums = ['d197eeb7fe5879dfbae789c459bcc901cb04d52c9cf5ef14fb07ff7a6b74560b']
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+]
+
+dependencies = [
+    ('Java', '11', '', SYSTEM),
+    ('pkgconf', '1.8.0'),
```
Comment (jfgrimm) [inline easybuild/easyconfigs/b/beagle-lib/beagle-lib-4.0.0-GCC-11.3.0.eb:23]: is `pkgconf` really a runtime dependency?

File: easybuild/easyconfigs/b/Beast/Beast-2.7.3-GCC-11.3.0.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/b/Beast/Beast-2.7.3-GCC-11.3.0.eb:20]: indeed, good catch

File: easybuild/easyconfigs/b/beagle-lib/beagle-lib-4.0.0-GCC-11.3.0.eb (hunk 2 above)
Comment (smoors) [reply easybuild/easyconfigs/b/beagle-lib/beagle-lib-4.0.0-GCC-11.3.0.eb:23]: good question, probably not, but this was just copied over from older easyconfigs so i'm not sure

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 5 out of 5 (2 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/ef75b27180fa186d416825891dda8cf4 for a full test report.

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
@boegelbot please test @ jsc-zen2

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @smoors !

**Pull request merged**